use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -0,0 +1,491 @@
+# Table of Contents
+
+- [Introduction](#introduction)
+  * [Helpful Links before starting](#helpful-links-before-starting)
+  * [Steps to enable Intel QAT in Kata Containers](#steps-to-enable-intel-qat-in-kata-containers)
+  * [Script variables](#script-variables)
+    + [Set environment variables (Every Reboot)](#set-environment-variables--every-reboot-)
+  * [Prepare the Host (Clear Linux)](#prepare-the-host--clear-linux-)
+    + [Identify which PCI Bus the Intel QAT card is on](#identify-which-pci-bus-the-intel-qat-card-is-on)
+    + [Install necessary bundles for Clear Linux and disable kernel updates](#install-necessary-bundles-for-clear-linux-and-disable-kernel-updates)
+    + [Replace Clear Linux kernel with LTS kernel](#replace-clear-linux-kernel-with-lts-kernel)
+    + [Allow insecure modules to load to host](#allow-insecure-modules-to-load-to-host)
+    + [Download Intel QAT drivers](#download-intel-qat-drivers)
+    + [Compile Intel QAT drivers against host kernel](#compile-intel-qat-drivers-against-host-kernel)
+    + [Copy Intel QAT configuration files and enable VF’s](#copy-intel-qat-configuration-files-and-enable-vf-s)
+    + [Expose and Bind Intel QAT virtual functions to VFIO-PCI (Every reboot)](#expose-and-bind-intel-qat-virtual-functions-to-VFIO-pci--every-reboot-)
+    + [Check Intel QAT virtual functions are enabled](#check-intel-qat-virtual-functions-are-enabled)
+  * [Prepare Kata Containers](#prepare-kata-containers)
+    + [Download Kata Kernel Source](#download-kata-kernel-source)
+    + [Build Kata Kernel](#build-kata-kernel)
+    + [Copy Kata Kernel](#copy-kata-kernel)
+    + [Prepare Kata root filesystem](#prepare-kata-root-filesystem)
+    + [Compile Intel QAT drivers for Kata Containers Kernel and add to Kata Containers rootfs](#compile-intel-qat-drivers-for-kata-containers-kernel-and-add-to-kata-containers-rootfs)
+    + [Copy Kata rootfs](#copy-kata-rootfs)
+    + [Update Kata configuration.toml to point to custom kernel and rootfs](#update-kata-configurationtoml-to-point-to-custom-kernel-and-rootfs)
+  * [Verify Intel QAT works in a Docker Kata Containers container](#verify-intel-qat-works-in-a-docker-kata-containers-container)
+  * [Build OpenSSL Intel QAT engine container](#build-openssl-intel-qat-engine-container)
+    + [Test Intel QAT in Docker](#test-intel-qat-in-docker)
+    + [Troubleshooting](#troubleshooting)
+  * [Optional Scripts](#optional-scripts)
+    + [Verify Intel QAT card counters are incremented](#verify-intel-qat-card-counters-are-incremented)
+    + [Unbind all VF’s from VFIO driver, reset, and rebind to the host](#unbind-all-vf-s-from-VFIO-driver--reset--and-rebind-to-the-host)
```

Comment:
Link doesn't work - I think s/vf-s/vfs/  fixes this one. Better check the others - and if this was generated with @jodh-intel tool, then we need to check how/why.

Reply:
He's not using https://github.com/kata-containers/tests/tree/master/cmd/check-markdown to generate the TOC currently as that only uses dashes for bullets.

Reply:
My next version will be from the check-markdown script. I had issues originally because the doc was created on windows.

Reply:
I think the `VF's` part of the link is still broken - one seems hyphenated, and the other not.. /cc @jodh-intel

Reply:
Yep - you may not have seen my comment but I think @eadamsintel may have forgotten to re-push his updated branch.